src/components/header/NavigationLink.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, ReactNode } from 'react';
+
+import { Language, useI18n } from '@smg-automotive/i18n-pkg';
+import { Badge } from '@chakra-ui/react';
+
+import { BreakpointName } from 'src/themes/shared/breakpoints';
+
+import Link from '../link';
+import Hide from '../hide';
+
+export const HeaderLink: FC<{
+  link: NavigationLinkProps;
+}> = ({ link }) => {
+  return (
+    <Hide below={link.showUnderMoreLinkBelow}>
+      <NavigationLink {...link} fontWeight="bold" />
+    </Hide>
+  );
+};
+
+export interface NavigationLinkProps {
+  url: Record<Language, string>;
+  translationKey: string;
+  isNew?: boolean;
+  isVisible?: boolean;
+  iconRight?: ReactNode;
+  showUnderMoreLinkBelow?: BreakpointName;
+  fontWeight?: 'regular' | 'bold';
```

Comment:
Do we really need these variations for the navigation link? Can we unify it as we have in a normal link? Can you double check it with the Designer? Then we can have one less prop